vendor-bin/php_codesniffer/composer.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
We can remove this - the `PHP_CODESNIFFER` standards are ones that we set within QA team for the Behat acceptance test code. They do not apply to other test code.